src/main/java/com/ericsson/gerrit/plugins/eiffel/listeners/ChangeMergedEventListener.java

```diff
@@ -64,6 +72,12 @@ protected void prepareAndSendEiffelEvent(Event gerritEvent,
         EiffelEventSender eiffelEventSender = new EiffelEventSender(pluginConfig);
         eiffelEventSender.setEiffelEventType(eiffelEvent.getClass().getSimpleName());
         eiffelEventSender.setEiffelEventMessage(eiffelEvent);
-        eiffelEventSender.send();
+        retryTemplate.execute(new RetryCallback<Void, HttpRequestFailedException>() {
+            @Override
+            public Void doWithRetry(RetryContext context) {
+                eiffelEventSender.send();
+                return null;
```

Comment:
Do we need to return null?

Reply:
This is a requirement of the doWithRetry method. I wont accept an implementation without an a null return. Has something to do with Void probably.